scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/EditProjectController.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import java.security.Principal;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.scriptoocloud.core.data.ProjectRepository;
+import edu.asu.diging.scriptoocloud.core.model.Project;
+import edu.asu.diging.scriptoocloud.core.model.impl.ProjectImpl;
+import edu.asu.diging.scriptoocloud.core.service.ProjectManager;
+
+@Controller
+public class EditProjectController {
+
+    @Autowired
+    private ProjectManager projectManager;
+    
+    private int projectID;
+
+    @RequestMapping(value = "/auth/edit/{id}", method = RequestMethod.POST)
+    public String post(@Valid @ModelAttribute("project") Optional<ProjectImpl> projectImpl, BindingResult result,
```

Comment:
should be a get request

Reply:
Do you have an example I may look at that utilizes a GET and Post? Currently, it POST's from the button press, then again when the user submits the information. I had some trouble understanding how I could do the same/similar thing ( have the user access edit.html with the projectID as a path variable) with a GET/POST exchange.